### ISSUE TITLE
Interpretation 

### ISSUE THREAD

**Issue by dermenes** [chaisemartinPackages/did_multiplegt_dyn#39]
Hi,

I'm unsure how to interpret the did_multiplegt_dyn results since they differ from other DID estimators. This discrepancy can be challenging for students like me. Can you give an example to how we can?

Thank you for your help.

**Comment by chaisemartinPackages** [comment]
Hi there,
This is Diego from Clément de Chaisemartin's RA Team.
Thanks for your interest in did_multiplegt_dyn!
did_multiplegt_dyn can be used in a wide variety of designs and settings. The assumptions underlying this command are described in the source paper [de Chaisemartin and D'Haultfoeuille (2024)](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3731856) and its functionalities are reviewed in the [companion paper](https://drive.google.com/file/d/1NGgScujLCCS4RrwdN-PC1SnVigfBa32h/view). These papers contain a detailed description of how did_multiplegt_dyn works and why it is different from other estimators. 
I guess that, in terms of comparisons, it may be overwhelming to perform a literature overview on your own. So, here you have some sources where you can read in simpler terms how to interpret did_multiplegt_dyn and how different commands compare to each other:
1. [Difference-in-Differences for Simple and Complex Natural Experiments, de Chaisemartin & D'Haultfoeuille](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4487202): it is a (free) working textbook about DiD that does a great job in terms of presenting the econometrics of did_multiplegt_dyn as well as comparing other estimators relative to each other (see Chapter 6.2);
2. [Two-Way Fixed Effects and Differences-in-Differences with Heterogeneous Treatment Effects: A Survey](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3980758): a review paper about the new DiD literature;
3. [Asjad Naqvi's website about DiD](https://asjadnaqvi.github.io/DiD/): a user-oriented website that describes all the new DiD estimators, I would suggest you to browse the [Resources](https://asjadnaqvi.github.io/DiD/docs/resources) section to access many tutorials and references.
Let me know if this is helpful.
Best,
Diego
